react-js/react.md

```diff
@@ -47,53 +50,65 @@ You will spend the day working through these materials.
 
 ### Topics to learn & Facilitate
 
-- What is React?
-  - React is a library for creating User Interfaces.
-  - This means it is used for creating what some refer to as the "view layer" of an application.
-- JSX adds to our JavaScript to allow for HTML-like markup in JavaScript.
-- Show Hello World example, plus expanded example.
-- Example for appending to a list in jQuery vs. the same in React.
+- React lets you generate HTML using JavaScript
+- The HTML can change over time, unlike regular HTML
+- No jQuery needed
+
+### Examples
+#### CodePen - Hello World component
+In this CodePen, a basic Hello World component is rendered.<br> 
+Try modifying line 5 of the JS file to see the changes made to the file rendered.
+<br>
+[Click Here](https://codepen.io/jamesjose03/pen/ExVjYBM)
```

Comment:
"Click here" is a [usability and accessibility anti-pattern](https://medium.com/@heyoka/dont-use-click-here-f32f445d1021) so please make the heading the link please

Reply:
👍